packages/reports/addon/components/navi-column-config.js

```diff
@@ -221,12 +226,35 @@ class NaviColumnConfig extends Component {
   }
 
   /**
-   * Stores element reference after render
+   * Opens a column
+   * @action
+   * @param {Object} column - The column to open
+   */
+  @action
+  openColumn(column) {
+    this.set('currentlyOpenFragment', column?.fragment);
+  }
+
+  /**
+   * Opens the date time column when it's the only column
+   * @action
+   */
+  @action
+  openDefaultColumn() {
+    const { columns, openColumn } = this;
+    if (columns.length === 1 && columns[0].type === 'timeDimension') {
```

Comment:
I think we have this condition in other places as well... can we save this refactor for when we have this new column? It will probably introduce other changes in the code